src/include/catalog/postgres/pg_proc.h

```diff
@@ -86,4 +90,6 @@ constexpr proc_oid_t NP_RUNNERS_DUMMY_REAL_PRO_OID = proc_oid_t(97);
 // SQL Function
 constexpr proc_oid_t DATE_PART_PRO_OID = proc_oid_t(101);
 
+constexpr proc_oid_t POSITION_PRO_OID = proc_oid_t(105);
```

Comment:
I would move this up with `CHARLENGTH_PRO_OID`

Reply:
Fixed